docs/experiments-plus/sequential-testing.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Sequential Testing
+sidebar_label: Sequential Testing
+slug: /experiments-plus/sequential-testing
+---
+
+## What is Sequential Testing?
+
+Traditional A/B testing best practices dictate that the readout of experiment metrics should occur only once, when the target sample size of the experiment has been reached.  Continous monitoring for the purpose of decision making results in inflated false positive rates, much higher than expected based on the significance level selected for the test.  
+
+This is because p-values fluctuate and are likely to drop in and out of significance just by random chance, even when there is no real effect.  Continuous monitoring introduces selection bias in the date we pick for the readout: Selectively choosing a date based on the observed results is essentailly cherry-picking a stat-sig results that would never be observed if the data were to be analyzed only over the entire, pre-determined durtaion of the experiment.
```

Comment:
```suggestion
This is because p-values fluctuate and are likely to drop in and out of significance just by random chance, even when there is no real effect.  Continuous monitoring introduces selection bias in the date we pick for the readout: Selectively choosing a date based on the observed results is essentially cherry-picking a stat-sig results that would never be observed if the data were to be analyzed only over the entire, pre-determined duration of the experiment.
```